.github/workflows/task-test.yml

```diff
@@ -214,7 +214,7 @@ jobs:
           LOG: 1
           CLEAR_LOGS: 0
           ENABLE_ASSERT: 1
-        run: make unit-tests
```

Comment:
why can't we maintain compatibility with `make` and `make unit-tests`?

Reply:
we probably should